MBBSEmu/HostProcess/ExportedModules/ExportedModuleBase.cs

```diff
@@ -34,7 +34,7 @@ public abstract class ExportedModuleBase
         ///     Pointers to files opened using FOPEN
         /// </summary>
         private protected readonly PointerDictionary<FileStream> FilePointerDictionary;
-        private protected readonly Dictionary<IntPtr16, BtrieveFileProcessor> BtrievePointerDictionaryNew;
+        private protected static readonly Dictionary<IntPtr16, BtrieveFileProcessor> BtrievePointerDictionaryNew = new Dictionary<IntPtr16, BtrieveFileProcessor>();
```

Comment:
this makes me incredibly sad. Can we create a shared dictionary and pass it into the constructor of each ExportedModuleBase subclass?

Reply:
I feel like this is probably the use case for static, when you want to share an object within a base class between implementing members. A private protected static in an abstract class is pretty encapsulated